src/Constraint.jl

```diff
@@ -40,10 +40,18 @@ end
 
 AbstractTrees.children(c::Constraint) = (c.child,)
 
-# A fallback. Define a new method if `MOI.Utilities.distance_to_set`
-# is not defined.
+# A fallback. Define a new method if `MOI.Utilities.distance_to_set` is not
+# defined.
 function is_feasible(x, set, tol)
-    return MOI.Utilities.distance_to_set(x, set) <= tol
+    try
+      return MOI.Utilities.distance_to_set(x, set) <= tol
+    catch
+        return true  # default to `true`
```

Comment:
Is this correct? Do we skip adding the constraint altogether or do we pass it to the solver who will mark it infeasible?

Reply:
Ooooo. This actually needs three-way logic